pkg/management/endpoint.go

```diff
@@ -699,6 +702,40 @@ func MakeUpdateAuthorizationsEndpoint(component Component) cs.Endpoint {
 	}
 }
 
+// MakePutAuthorizationEndpoint creates an endpoint for PutAuthorization
+func MakePutAuthorizationEndpoint(component Component) cs.Endpoint {
```

Comment:
MakeAddAuthorizationEndpoint

Reply:
Corrected in commit e65d1abb21021efc38ddf4adf3ef93c6273bd283